modules/apps/frontend-theme/frontend-theme-dialect/src/css/dialect/variables/_buttons.scss

```diff
@@ -123,3 +123,15 @@ $btn-link: (
 		color: var(--color-action-primary-disabled),
 	),
 );
+
+$btn-colors: (
+	color: $white,
+	hover: (
+		color: $white,
+	),
+);
+
+$btn-danger: $btn-colors;
```

Comment:
We can unset the color property with
```
(
    color: c-unset,
    hover: (
        color: c-unset,
    ),
);
```

Reply:
@ilzamcmed it will, but Dialect sets colors on their button variants.

Reply:
@pat270 
I've tested on Dialect sample and it does not affect other variants. 
I also added pseudo-classes that affect the color of the buttons.